### ISSUE TITLE
Installer crashing out because not finding cnd files.

### ISSUE THREAD

**Issue by NekoJonez** [thekovic/Indy3DModInstaller#3]
https://github.com/thekovic/Indy3DModInstaller/blob/b594de5d98fcc1de08c45889d481abc19c766dd5/Indy3DModInstaller/Program.cs#L122

A strange issue here, but the gobext tool is used on my machine... It doesn't create a the seperate folders BUT it creates three folders named CD1_GOB, CD2_GOB & JONES3D_GOB. 

And I need to merge them.... So, the ndy folder isn't present and it needs to merge first. 

--> That's why the installer can crash out here. 

Does the gobext.exe tool have different behaviour ?

**Comment by thekovic** [comment]
Yeah, so I ship Indy3DModInstaller with a custom-made build of gobext.exe because I needed to patch the output directory option (see https://github.com/smlu/Urgon/pull/11 ). When you replaced it with the standard one, you broke the expected flow of the program.

As for why the intended gobext.exe doesn't work ( https://github.com/thekovic/Indy3DModInstaller/issues/2 ), I don't understand why it compiled for that C++ runtime. I'll see if I can do something about that.

**Comment by NekoJonez** [comment]
Ah thats makes a ton of sense. 
This puts me a bit at ease for my powershell script. But I am going to build in some custom code if your pr ever gets merged.
